backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/CreateExperimentRunHandler.java

```diff
@@ -67,26 +67,8 @@ public CreateExperimentRunHandler(
   }
 
   public InternalFuture<ExperimentRun> createExperimentRun(final CreateExperimentRun request) {
-    // Validate arguments
-    var futureTask =
-        InternalFuture.runAsync(
-            () -> {
-              if (request.getProjectId().isEmpty()) {
-                throw new InvalidArgumentException(
-                    "Project ID not found in CreateExperimentRun request");
-              } else if (request.getExperimentId().isEmpty()) {
-                throw new InvalidArgumentException(
-                    "Experiment ID not found in CreateExperimentRun request");
-              }
-            },
-            executor);
```

Comment:
Moved to DAO level at https://github.com/VertaAI/modeldb/pull/2244/files#diff-f554e4fd8e5f43fa49db0fa2ba8f6e7a5903f2259b01a1a1e411c21189a2ce78R1032-R1044